webpack.config.js

```diff
@@ -2,19 +2,22 @@
 
 require('babel-polyfill'); // Define Object.assign() from ES6 in ES5.
 
+var webpack = require('webpack');
 var HasteResolverPlugin = require('haste-resolver-webpack-plugin');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var process = require('process');
-var webpack = require('webpack');
```

Comment:
Maybe it was moved after rebase. You and me introduced this changes in the file, so I removed one of the lines.